format.go

```diff
@@ -35,6 +35,8 @@ func Format(v interface{}) string {
 		return "s" + quoteString(s)
 	case string:
 		return quoteString(v)
+	case uintptr, uint, uint8, uint16, uint32, uint64:
```

Comment:
What do you think about adding a comment explaining why we are special casing uint types?

Reply:
I lack the appropriate words. 
would something like the following suffice?
`special cased so that we can display the type and the value in decimal notation`

Reply:
`// Use decimal base (rather than hexadecimal) for representing uint types.` ?

Reply:
fixed in https://github.com/frankban/quicktest/pull/93/commits/4e0e208393f58f503352083eb65a626186bf32fd